Lesson 3

Aleksa-esme opened the pull request.

Aleksa-esme committed:
created new tables for DB

Aleksa-esme committed:
added models in admin pannel

vprinz commented:
Отвечаю на ваши вопросы:
1. Нужно включить поддержку Django в проекте https://www.jetbrains.com/help/pycharm/django-support7.html#enable-support или проверьте, что вы через PyCharm открыли именно сам проект (как мы это делали на 1ом уроке)
2. Так у вас функция str находится не в классе вообще. Посмотрите внимательнее код с урока и ваш

Aleksa-esme committed:
added models in admin pannel
added media files
displaying the first four products in for-loop on index.html page

Aleksa-esme committed:
added url namespaces

Aleksa-esme committed:
added fixtures

Aleksa-esme commented:
> Отвечаю на ваши вопросы:
> 
> 1. Нужно включить поддержку Django в проекте https://www.jetbrains.com/help/pycharm/django-support7.html#enable-support или проверьте, что вы через PyCharm открыли именно сам проект (как мы это делали на 1ом уроке)
> 2. Так у вас функция str находится не в классе вообще. Посмотрите внимательнее код с урока и ваш

Исправила

vprinz commented on gamesshop/settings.py:
```diff
@@ -121,7 +121,11 @@
 STATIC_URL = '/static/'
 
-STATICFILES_DIRS = (os.path.join(BASE_DIR, "static")),
+STATICFILES_DIRS = (os.path.join(BASE_DIR, "static"),
```
Используйте везде один стиль кавычек

Aleksa-esme replied:
Исправила

Aleksa-esme committed:
changed quotation marks
